Features/Content/place.feature

```diff
@@ -11,3 +18,11 @@ Feature: Place
     Scenario: As a visitor, I can see the place list sorted
         Given I am on "/placelist/110/45.6152665/5.2234365/10000"
         Then I see "Mount Ventoux" text
+
+    Scenario: As a visitor, I can see the Mount Fuji page
+        Given I am on "/eZ-Mountains/Mount-Fuji"
+        Then I see "Mount Fuji" text
+        And I see "Mount Fuji, located on Honshu Island, is the highest mountain in Japan at 3,776.24 m (12,389 ft)." text
+        And I see the thumbnail for image with alternative text "mount fuji"
+        And Path for thumbnail image with alternative text "mount fuji" should match "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg"
+        And File "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg" exists
```

Comment:
@clash82 some context:
the argument above is me and @bdunogier argueing over having the path in the test implementation, aka the Context. But having the path in the *Spec*, aka .feature file, is a whole other level. Please take this to heart: *BDD is a Specification by example system, it is not a test framework*, so what you are writing here is a specification for how the system should work, and the feature you are testing is placement, not the internals of the image system.